README.md

```diff
@@ -1107,6 +1107,7 @@ OS - OpenSource
 - [rdf.sh](https://github.com/seebi/rdf.sh) - A multi-tool shell script for doing Semantic Web jobs on the command line.
 - [QuitStore](https://github.com/AKSW/QuitStore) - Quads in Git - Distributed Version Control for RDF Knowledge Bases.
 - [QuitDiff](https://github.com/AKSW/QuitDiff) - Git diff into SparQL / Eccrev vocabulary.
+- [OntoGen](https://ontogen.io/) - A RDF aware version control system.
```

Comment:
The correct spelling is "Ontogen", which is especially important since there is another (albeit unrelated) system with this name.

Reply:
Fixed the spelling and also changed the description.